src/main/java/com/google/sps/servlets/BooksServlet.java

```diff
@@ -88,10 +89,29 @@ private String encodeBookMapAsJson(LinkedHashMap<String, List<Book>> booksMap) {
       JsonObject bookJson = booksJsonArray.get(i).getAsJsonObject().getAsJsonObject("volumeInfo"); 
       String title = bookJson.get("title").getAsString();
       String link = bookJson.get("infoLink").getAsString();
-      String image = bookJson.getAsJsonObject("imageLinks").get("thumbnail").getAsString();
-      String description = bookJson.get("description").getAsString();
-      String author = formatAuthors(bookJson.getAsJsonArray("authors"));
-      books.add(new Book.Builder(title, link).withImage(image).withDescription(description).withAuthor(author).build());
+      Book.Builder bookBuilder = new Book.Builder(title, link);
```

Comment:
I don't know why this showed up in this PR because I did this in a previous PR, but the code is fine.

Reply:
Same for below

Reply:
If you merged that PR into MVPTEST before MVPTEST is merged into master the changes will appear in this PR.

Reply:
My bad, thought I merged it into master but you're right.